pkg/test/environment.go

```diff
@@ -642,6 +642,39 @@ func (e *Environment) StartPrometheus(opniAgentPort int, override ...*overridePr
 	return port
 }
 
+func (e *Environment) StartAlertManager(ctx context.Context, configFile string) (webPort, apiPort int) {
+
+	webPort, err := freeport.GetFreePort()
```

Comment:
You can use the helper function `GetFreePorts(2)`

Reply:
Done